RSDKv5/RSDK/Input/Wii/WiiInputDevice.cpp

```diff
@@ -6,20 +6,61 @@ void RSDK::SKU::InputDeviceWii::UpdateInput() {
+            break;
+        case WPAD_EXP_NUNCHUK:
+            this->stateUp       = (this->buttonMasks & WPAD_BUTTON_UP) != 0;
+            this->stateDown     = (this->buttonMasks & WPAD_BUTTON_DOWN) != 0;
+            this->stateLeft     = (this->buttonMasks & WPAD_BUTTON_LEFT) != 0;
+            this->stateRight    = (this->buttonMasks & WPAD_BUTTON_RIGHT) != 0;
+            this->stateA        = (this->buttonMasks & WPAD_BUTTON_B) != 0;
+            this->stateB        = (this->buttonMasks & WPAD_BUTTON_A) != 0;
+            this->stateC        = (this->buttonMasks & 0) != 0;
+            this->stateX        = (this->buttonMasks & WPAD_NUNCHUK_BUTTON_C) != 0;
+            this->stateY        = (this->buttonMasks & WPAD_NUNCHUK_BUTTON_Z) != 0;
+            this->stateZ        = (this->buttonMasks & 0) != 0;
+            this->stateStart    = (this->buttonMasks & WPAD_BUTTON_PLUS) != 0;
+            this->stateSelect   = (this->buttonMasks & WPAD_BUTTON_MINUS) != 0;
+            this->stateUp       |= (data->exp.nunchuk.js.pos.y > data->exp.nunchuk.js.center.y + 5) ? 1 : 0;
+            this->stateDown     |= (data->exp.nunchuk.js.pos.y < data->exp.nunchuk.js.center.y - 5) ? 1 : 0;
+            this->stateLeft     |= (data->exp.nunchuk.js.pos.x < data->exp.nunchuk.js.center.x - 5) ? 1 : 0;
+            this->stateRight    |= (data->exp.nunchuk.js.pos.x > data->exp.nunchuk.js.center.x + 5) ? 1 : 0;
+            break;
+        case WPAD_EXP_CLASSIC:
+            this->stateUp       = (this->buttonMasks & WPAD_CLASSIC_BUTTON_UP) != 0;
+            this->stateDown     = (this->buttonMasks & WPAD_CLASSIC_BUTTON_DOWN) != 0;
+            this->stateLeft     = (this->buttonMasks & WPAD_CLASSIC_BUTTON_LEFT) != 0;
+            this->stateRight    = (this->buttonMasks & WPAD_CLASSIC_BUTTON_RIGHT) != 0;
+            this->stateA        = (this->buttonMasks & WPAD_CLASSIC_BUTTON_B) != 0;
+            this->stateB        = (this->buttonMasks & WPAD_CLASSIC_BUTTON_A) != 0;
+            this->stateC        = (this->buttonMasks & 0) != 0;
+            this->stateX        = (this->buttonMasks & WPAD_CLASSIC_BUTTON_Y) != 0;
+            this->stateY        = (this->buttonMasks & WPAD_CLASSIC_BUTTON_X) != 0;
+            this->stateZ        = (this->buttonMasks & 0) != 0;
+            this->stateStart    = (this->buttonMasks & WPAD_CLASSIC_BUTTON_PLUS) != 0;
+            this->stateSelect   = (this->buttonMasks & WPAD_CLASSIC_BUTTON_MINUS) != 0;
+            this->stateUp       |= (data->exp.classic.ljs.pos.y > data->exp.classic.ljs.center.y + 5) ? 1 : 0;
+            this->stateDown     |= (data->exp.classic.ljs.pos.y < data->exp.classic.ljs.center.y - 5) ? 1 : 0;
+            this->stateLeft     |= (data->exp.classic.ljs.pos.x < data->exp.classic.ljs.center.x - 5) ? 1 : 0;
+            this->stateRight    |= (data->exp.classic.ljs.pos.x > data->exp.classic.ljs.center.x + 5) ? 1 : 0;
+            break;
+        default:
+            break;
```

Comment:
Consider moving `default` up to the `WPAD_EXP_NONE` case.

Reply:
`default` has been moved up to the `WPAD_EXP_NONE` case.